from chat.exceptions.not_provided_exception import (
    NotProvidedException as NotProvidedException
)
from chat.exceptions.not_found_exception import (
    NotFoundException as NotFoundException
)